src/main/scala/it/pagopa/interop/tenantmanagement/api/impl/TenantApiServiceImpl.scala

```diff
@@ -127,7 +127,7 @@ class TenantApiServiceImpl(
       )
     } yield ()
 
-    onComplete(result) { deleteTenantResponse[Unit](operationLabel)(_ => deleteTenant204) }
+    onComplete(result) { deleteTenantResponse[Unit](operationLabel)(_ => addTenantMail204) }
```

Comment:
As per previous comment
```suggestion
    onComplete(result) { addTenantMailResponse[Unit](operationLabel)(_ => addTenantMail204) }
```